src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categorySlice = createSlice({
+  name: 'categories',
+  initialState: {
+    categories: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    getCategoryStatus: (state) => state.status,
```

Comment:
- [ ] I will recommend you do a check before adding the status, in fact, the status reducer should have a payload, it can be as follow:
```
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```